src/components/CheckoutPayment/__tests__/formOptions.test.js

```diff
@@ -0,0 +1,25 @@
+import { months, nextYearsFromDate, monthsOptions, yearsOptions } from '../formOptions';
+
+describe('FUNC - months', () => {
+  it('generates list of 12 months', () => {
+    expect(months()).toMatchSnapshot();
```

Comment:
Since it's generates static seriazable data snapshot is perfect candidate for test here.
Classic assertion would look like kind of stupid: `expect(months()).toEqual([1,2,...,12]);`